app/components/views/Home/MobileMenu/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useState } from "react";
+import { Link } from "react-router-dom";
+import { IconButton, Menu, MenuItem, Zoom } from "@mui/material";
```

Comment:
Please use this import syntax, which iirc has better tree shaking.

```ts
import IconButton from '@mui/material/IconButton';
```

Reply:
done